python-packages/securesync/devices/decorators.py

```diff
@@ -0,0 +1,30 @@
+"""
+This sub-module deals with Zones--how they're defined, permissions related to them,
+and the functions to allow joining them.
+
+The "engine" sub-module deals with sharing data within a zone.
+"""
```

Comment:
This docstring looks like it belongs back in the `__init__` from whence it came? (it describes the module, not the function here)